tui/src/root.rs

```diff
@@ -0,0 +1,69 @@
+use ratatui::{
+    backend::CrosstermBackend,
+    crossterm::event::{self, Event, KeyCode, KeyEvent},
+    layout::{Alignment, Constraint, Layout},
+    style::{Style, Stylize},
+    widgets::{Paragraph, Wrap},
+    Terminal,
+};
+use std::io;
+
+pub fn check_root(terminal: &mut Terminal<CrosstermBackend<io::Stdout>>) -> io::Result<bool> {
+    if sudo::check() == sudo::RunningAs::Root {
+        terminal.draw(|frame| {
+            let root_warn = Paragraph::new(
+                r#"
+!!!!!!!!!!!!!! YOU ARE ABOUT TO RUN LINUTIL AS ROOT !!!!!!!!!!!!!!
+This utility prioritizes compatibility with non-root environments.
+Some scripts may work without any issues, some may not.
+You have been warned!
```

Comment:
i think we have enough exclamation points

```suggestion
You have been warned.
```

Reply:
Lmao

Reply:
Exclamation looks good